config_center/apollo/mockDubbog.properties.json

```diff
@@ -0,0 +1 @@
+{"appId":"testApplication_yang","cluster":"default","namespaceName":"mockDubbog.properties","releaseKey":"20191104105242-0f13805d89f834a4","configurations":{"registries.hangzhouzk.username":"11111"}}
```

Comment:
this file is generated by apollo test case. you can remove it , i will delete it when testcase finished